### ISSUE TITLE
/usr/bin/crawl Permission denied

### ISSUE THREAD

**Issue by bricas** [webrecorder/browsertrix-crawler#502]
I just happened to pull the latest image (1.0.0) and using the usual crawl command i got the error:

```
argv0-ignore: line 1: /usr/bin/crawl: Permission denied
```

**Comment by ikreymer** [comment]
What platform are you testing on? What container runtime are you using?
Weren't not able to repro this, the amd64 and arm64 versions appear to run fine.

**Comment by bricas** [comment]
```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.10.4
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.17.3
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 2
  Running: 1
  Paused: 0
  Stopped: 1
 Images: 2
 Server Version: 23.0.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 2806fc1057397dbaeefbea0e4e17bddfbd388f38
 runc version: v1.1.5-0-gf19387a
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.15.0-100-generic
 Operating System: Ubuntu 22.04.4 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 15.61GiB
 Name: [redacted]
 ID: LP32:KDHB:SEQR:UMCI:H5VW:DHJB:3P52:G7DN:5IXD:ZDVL:L6HH:72YK
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

**Comment by bricas** [comment]
I can try more stuff later, currently in the middle of a craw using the `webrecorder/browsertrix-crawler:0.12.4` tag which works fine.

**Comment by ikreymer** [comment]
I found the issue - it is because the `/crawls` directory (whichever directory you're mapping for crawls) is owned by a non-root user. Unfortunately, that seems to have broken in 1.0.0. Will do a patch shortly. In the meantime, if you point it to a root-owned directory (or one that doesn't exist, it'll create a new one), it should work.

**Comment by bricas** [comment]
Can confirm this fixes it. Thank you!

**Comment by ikreymer** [comment]
Released in 1.0.1, image should be available shortly, let us know if there are any issues!
